pkg/cluster/filesystems.go

```diff
@@ -0,0 +1,45 @@
+package cluster
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/zalando-incubator/postgres-operator/pkg/spec"
+	"github.com/zalando-incubator/postgres-operator/pkg/util/constants"
+	"github.com/zalando-incubator/postgres-operator/pkg/util/filesystems"
+)
+
+func (c *Cluster) getPostgresFilesystemInfo(podName *spec.NamespacedName) (err error, device string, fstype string) {
```

Comment:
The first is obviously a blunder, but the second I would do nothing about. I find it less error-prone to name multiple consecutive arguments of the same type in a declaration, even though the names are not used to return values.